src/i18n/en.js

```diff
@@ -150,24 +150,26 @@ You'll find an Email with steps to follow.`,
     plan_suscribers: 'Suscribers',
   },
   invoices_list: {
-    account_id_column: `Account Id`,
     amount_column: `Amount`,
     balance_column: `Balance`,
     control_panel_account_preferences_url: `${urlControlPanelMain}?section=AccountPreferences`,
     control_panel_billing_information_section: `Billing Information`,
     control_panel_billing_information_url: `${urlControlPanel}/AccountPreferences/BillingInformationSettings`,
     control_panel_section: `Control Panel`,
     creation_date_column: `Creation Date`,
+    credit_note_text: `Credit Note`,
```

Comment:
is not the suffix `_text` redundant?

Reply:
With the change from above this would be obsolete.

Reply:
Done